ui/component/walletBackup/view.jsx

```diff
@@ -1,6 +1,5 @@
 // @flow
```

Comment:
This file is another case of "partial removal", which leaves it in a limbo state -- it's technically unused, but it's also not a complete reference.  e.g. if we want to support exporting wallet in Web one day, we'll have to look at Git to see where the partially missing items are to provide a Web-equivalent.

For these cases where it is unused in Web, we should try to maintain things as a "complete unit":  either leave as-is, or delete entirely.

Was this file causing build issues?  I was thinking it is not being compiled since the only place that imported it was gated by `App`, so we can leave as is (or delete) + remove the places that imported it.